examples/vision/knowledge_distillation.py

```diff
@@ -104,10 +104,20 @@ def train_step(self, data):
 
             # Compute losses
             student_loss = self.student_loss_fn(y, student_predictions)
+
+            """ Compute scaled loss from https://arxiv.org/abs/1503.02531
+                Since the magnitudes of the gradients produced by the soft
```

Comment:
Please turn this into a comment (use `#`)

Reply:
Thanks for checking! Fixed now (also in `.ipynb` and `.md`).